api/package.json

```diff
@@ -3,8 +3,12 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
+    "api:dev": "nodemon --watch 'api/vudl.ini' --watch 'api/**/*.js' --watch 'api/**/*.ejs' --ignore api/node_modules/ src/app.ts",
```

Comment:
Should this be `start:dev` (or should `start` be renamed to `api`) to keep a parallel structure between dev and prod tasks? Also, does the current `start` task actually work/do anything? I suspect it may be a relic of boilerplate setup code.

Reply:
Made change to start:api, and start:queue